api/src/test/java/org/apache/iceberg/TestHelpers.java

```diff
@@ -559,4 +561,21 @@ public List<Long> splitOffsets() {
       return null;
     }
   }
+
+  public static class TestMetricsReporter implements MetricsReporter {
```

Comment:
can we also move this to the respective test class? I don't think it's necessary to put this into `TestHelpers`

Reply:
done